src/lib.rs

```diff
@@ -158,6 +200,31 @@ impl<T: PrimInt + Unsigned> SparseSet<T> {
     }
 }
 
+/// A macro to create and initialize sets in one go.
+///
+/// ```rust
+/// use thinset::{set, SparseSet};
+/// let mut set: SparseSet<u32> = set!(4, 32, 16, 24, 63);
```

Comment:
Does this work with square brackets? I think that's the more conventional style, see e.g. https://doc.rust-lang.org/std/macro.vec.html
```suggestion
/// let mut set: SparseSet<u32> = set![4, 32, 16, 24, 63];
```

Reply:
Yes, all kinds of brackets work. I think they can be chosen at the call site. I felt like using square brackets suggested that one is initializing an ordered structure.